core/templates/pages/story-editor-page/editor-tab/story-node-editor.component.html

```diff
@@ -117,31 +117,47 @@
         <div class="e2e-test-add-chapter-outline">
           <div class="story-node-header oppia-mobile-collapsible-card-header" (click)="toggleChapterOutline()">
             <span>Chapter Outline</span>
-            <i class="fa fa-caret-down"
-               *ngIf="!chapterOutlineIsShown"
-               aria-hidden="true">
-            </i>
-            <i class="fa fa-caret-up"
-               *ngIf="chapterOutlineIsShown"
-               aria-hidden="true">
-            </i>
           </div>
-          <div class="chapter-card-inputs oppia-mobile-collapsible-card-content" *ngIf="chapterOutlineIsShown">
+          <br>
+          <div class="outline-Finalized-container" *ngIf="oldOutline">
             Outline Finalized
             <i *ngIf="!outlineIsFinalized" (click)="finalizeOutline()" class="far fa-square md-18 e2e-test-finalize-outline" aria-label="Finalize outline"></i>
             <i *ngIf="outlineIsFinalized" (click)="unfinalizeOutline()" class="fas fa-check-square md-18" aria-label="Unfinalize outline"></i>
-            <div tabindex="-1" (blur)="toggleChapterOutlineButtons()" (click)="toggleChapterOutlineButtons()">
+          </div>
+          <div *ngIf="!chapterOutlineIsShown">
+            <div class="outline-content-container" *ngIf="!oldOutlineToReadOnly">
+              <input class="chapter-outline-editor-content e2e-test-node-outline-editor-content e2e-test-rte"
+                     readonly
+                     (click)="chapterOutlineIsShown=true"
+                     placeholder="Open chapter outline editor.">
+              <label class="fas fa-pencil-alt oppia-icon" (click)="chapterOutlineIsShown=true"></label>
+            </div>
+            <div class="outline-content-container" *ngIf="oldOutlineToReadOnly" >
```

Comment:
See above re naming, please also remove the space before ">"

Reply:
I replace `oldOutlineToReadOnly` with `outlineEditViewIsShown`. Then I remove the space.
![fix_html_2](https://github.com/oppia/oppia/assets/140389350/b5dc7dda-9c9d-4fca-8e84-3ff08025abe4)